src/Files.App/Shell/Win32API.cs

```diff
@@ -805,12 +805,19 @@ public static async Task<bool> InstallInf(string filePath)
 			}
 		}
 
-		public static void InstallFont(string fontFilePath)
+		public static void InstallFont(string fontFilePath, bool forAllUsers)
```

Comment:
Are these changes related to the rich command? If not, we should limit the scope of this PR.